README.md

```diff
@@ -1,12 +1,12 @@
-# SSP EKS Workloads
+# EKS Blueprints Workloads
 
-Welcome to the SSP EKS Workloads repository.
+Welcome to the EKS Blueprints Workloads repository.
 
-This repository serves as a sample GitOps configuration repository that is meant to be used with the [Amazon EKS SSP Quick Start](https://github.com/aws-quickstart/quickstart-ssp-amazon-eks).
+This repository serves as a sample GitOps configuration repository that is meant to be used with the [Amazon EKS Blueprints Quick Start](https://github.com/aws-quickstart/cdk-eks-blueprints).
 
 ## Documentation  
 
-Please refer to the Amazon EKS SSP Quick Start [Getting Started](https://aws-quickstart.github.io/ssp-amazon-eks/getting-started/) guide for details on how to bootstrap an EKS cluster with the workload configuration contained in this repository.
+Please refer to the Amazon EKS Blueprints Quick Start [Getting Started](https://aws-quickstart.github.io/ssp-amazon-eks/getting-started/) guide for details on how to bootstrap an EKS cluster with the workload configuration contained in this repository.
```

Comment:
this link is not rendering https://aws-quickstart.github.io/ssp-amazon-eks/getting-started/